www/docs/en/dev/guide/next/index.md

```diff
@@ -23,85 +23,62 @@ description: A look at topics that new Cordova developers will encounter.
 
 **Additional Performance Articles and Resources**
 
-* ["You half assed it"](http://sintaxi.com/you-half-assed-it)
 * ["Top Ten Performance Tips for PhoneGap and Hybrid Apps"](http://coenraets.org/blog/2013/10/top-10-performance-techniques-for-phonegap-and-hybrid-apps-slides-available/)
-* ["Fast Apps and Sites with JavaScript"][1]
-
- [1]: https://channel9.msdn.com/Events/Build/2013/4-313
+* ["Fast Apps and Sites with JavaScript"](https://channel9.msdn.com/Events/Build/2013/4-313)
 
 ## 3) Recognize and Handle Offline Status
 
 See the previous tip about networks. Not only can you be on a slow network, it is entirely possible for your application to be completely offline. Your application should handle this in an intelligent manner. If your application does not, people will think your application is broken. Given how easy it is to handle (Cordova supports listening for both an offline and online event), there is absolutely no reason for your application to not respond well when run offline. Be sure to test (see the Testing section below) your application and be sure to test how your application handles when you start in one state and then switch to another.
 
-Note that the online and offline events, as well as the Network Connection API is not perfect. You may need to rely on using an XHR request to see if the device is truly offline or online. At the end of the day, be sure add some form of support for network issues - in fact, the Apple store (and probably other stores) will reject apps that don’t properly handle offline/online states. For more discussion on this topic, see
-["Is This Thing On?"](http://blogs.telerik.com/appbuilder/posts/13-04-23/is-this-thing-on-%28part-1%29)
+Note that the online and offline events, as well as the Network Connection API, is not perfect. You may need to rely on using an XHR request to see if the device is truly offline or online. At the end of the day, be sure add some form of support for network issues - in fact, the Apple store (and probably other stores) will reject apps that don't properly handle offline/online states. For more discussion on this topic, see
+["Is This Thing On?"](https://blogs.telerik.com/appbuilder/posts/13-04-23/is-this-thing-on-%28part-1%29)
 
 # Handling Upgrades
 
 ## Upgrading Cordova Projects
 
-If your existing project was created using Cordova 3.x, you can upgrade the project by issuing the following:
+You can upgrade the project by running the following:
```

Comment:
I don't know that this command actually works... I seem to recall that it either renames the platform directory and installs a newer version beside it, or that it tries to actually upgrade the platforms files and fails horribly.